src/silx/gui/data/DataViews.py

```diff
@@ -1074,30 +1073,25 @@ def createWidget(self, parent):
         widget.toolBar().addAction(self.__aggregationModeAction)
         self.__aggregationModeAction.sigAggregationModeChanged.connect(self._aggregationModeChanged)
 
+        self.__imageItem = ImageDataAggregated()
+        self.__imageItem.setAggregationMode(self.__aggregationModeAction.getAggregationMode())
+        self.__imageItem.setName("data")
+        self.__imageItem.setColormap(widget.getDefaultColormap())
+        widget.addItem(self.__imageItem)
+        widget.setActiveImage(self.__imageItem)
+
         widget.setKeepDataAspectRatio(True)
         widget.getXAxis().setLabel("X")
         widget.getYAxis().setLabel("Y")
         maskToolsWidget = widget.getMaskToolsDockWidget().widget()
         maskToolsWidget.setItemMaskUpdated(True)
         return widget
-    
-    def getAggregationModeAction(self) -> AggregationModeAction:
-        """Action toggling the aggregation mode action
-        """
-        return self.__aggregationModeAction
```

Comment:
No need of a method for it: it is only used internally and `self.__aggregationModeAction` is not always available since it is not set in `__init__`